### PR TITLE
17 system of display textimage

### PR DESCRIPTION
**Pull request by AxelF44**
Implemented system to display Text and Image !

### CODE REVIEW

File: include/Object/Image.hpp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+         *
+         * @return sf::Vector2f Position X - Y
+         */
+        sf::Vector2f getPos(void);
+
+    private:
+
+        sf::Texture     _texture;
+        sf::Vector2f    _position;
+};
```
Comment (Marius-P1) [inline include/Object/Image.hpp:60]:
```suggestion
};

```

File: include/Object/Text.hpp [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+         */
+        sf::Vector2f getPos(void);
+
+    private:
+
+        sf::Font        _font;
+        sf::String      _content;
+        unsigned int    _charactereSize;
+        sf::Vector2f    _position;
+};
```
Comment (Marius-P1) [inline include/Object/Text.hpp:84]:
```suggestion
};

```

File: include/Object/Image.hpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+        /**
+         * @brief Get the Texture object
+         *
+         * @return sf::Texture
+         */
+        sf::Texture getTexture(void);
+
+        /**
+         * @brief Get the Pos object
```
Comment (Njord201) [inline include/Object/Image.hpp:50]:
```suggestion
         * @brief Get the Pos object.
```

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

File: include/Object/Text.hpp [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+class FlameClicker::Text {
+
+    public:
+
+        /**
+         * @brief Construct a new Text object.
+         *
+         */
+        Text(std::string pathFont, sf::String content, unsigned int charactereSize,
```
Comment (Njord201) [inline include/Object/Text.hpp:34]: Describe parameters

File: include/Object/Text.hpp [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+         * @return sf::Font font of Text
+         */
+        sf::Font getFont(void);
+
+        /**
+         * @brief Set the String object.
+         *
+         * @return sf::String string to set
+         */
+        void setString(sf::String content);
```
Comment (Njord201) [inline include/Object/Text.hpp:55]: Check your documentation

File: include/Object/Text.hpp [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+        /**
+         * @brief Get the String object.
+         *
+         * @return sf::String content to display
+         */
+        sf::String getString(void);
```
Comment (Njord201) [inline include/Object/Text.hpp:62]: Check your documentation

File: src/Object/Image.cpp [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+#include "Object/Image.hpp"
+
+FlameClicker::Image::Image(std::string pathTexture, sf::Vector2f position)
+{
+    if (!this->_texture.loadFromFile(pathTexture))
+        std::cout << "File doesn't exist" << std::endl;
+    this->_position = position;
+}
+
+sf::Texture FlameClicker::Image::getTexture(void)
```
Comment (Njord201) [inline src/Object/Image.cpp:17]:
```suggestion
sf::Texture FlameClicker::Image::getTexture(void) const
```

File: src/Object/Image.cpp [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+{
+    return this->_texture;
+}
+
+sf::Vector2f FlameClicker::Image::getPos(void)
```
Comment (Njord201) [inline src/Object/Image.cpp:22]:
```suggestion
sf::Vector2f FlameClicker::Image::getPos(void) const
```

File: src/Object/Text.cpp [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+unsigned int charactereSize, sf::Vector2f position)
+{
+    if (!this->_font.loadFromFile(pathFont))
+        std::cout << "File doesn't exist" << std::endl;
+    this->_content = content;
+    this->_charactereSize = charactereSize;
+    this->_position = position;
+}
+
+sf::Font FlameClicker::Text::getFont(void)
```
Comment (Njord201) [inline src/Object/Text.cpp:20]:
```suggestion
sf::Font FlameClicker::Text::getFont(void) const
```

File: src/Object/Text.cpp [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+{
+    return this->_font;
+}
+
+sf::String FlameClicker::Text::getString(void)
```
Comment (Njord201) [inline src/Object/Text.cpp:25]:
```suggestion
sf::String FlameClicker::Text::getString(void) const
```

File: src/Object/Text.cpp [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+{
+    return this->_content;
+}
+
+unsigned int FlameClicker::Text::getCharactereSize(void)
```
Comment (Njord201) [inline src/Object/Text.cpp:30]:
```suggestion
unsigned int FlameClicker::Text::getCharactereSize(void) const
```

File: src/Object/Text.cpp [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+{
+    return this->_charactereSize;
+}
+
+sf::Vector2f FlameClicker::Text::getPos(void)
```
Comment (Njord201) [inline src/Object/Text.cpp:35]:
```suggestion
sf::Vector2f FlameClicker::Text::getPos(void) const
```

**Review by Njord201 — CHANGES_REQUESTED** [review]

**Review by Njord201 — CHANGES_REQUESTED** [review]
You have conflict with main branch

File: include/Object/Text.hpp [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+     */
+   class Text;
+};
+
+class FlameClicker::Text {
+
+    public:
+
+        /**
+         * @brief Construct a new Text object
```
Comment (Njord201) [inline include/Object/Text.hpp:31]:
```suggestion
         * @brief Construct a new Text object.
```

**Review by Njord201 — APPROVED** [review]

**Review by mathieurobert1 — APPROVED** [review]

**Review by Marius-P1 — APPROVED** [review]
:ok_hand:

**Review by theophile-jr — APPROVED** [review]
god job axel

**Pull request merged**